docs/pages/database-access/guides/azure-postgres-mysql.mdx

```diff
@@ -27,101 +25,128 @@ This guide will help you to:
 
 (!docs/pages/includes/database-access/token.mdx!)
 
 Install Teleport on the host where you will run the Teleport Database Service:
 
 (!docs/pages/includes/install-linux.mdx!)
 
-Create the Database Service configuration, specifying a region like this:
-
-<ScopedBlock scope={["oss", "enterprise"]}>
+Create the Database Service configuration.
 
 <Tabs>
 <TabItem label="PostgreSQL">
 
+- Specify the region for your database(s) in `--azure-postgres-discovery`.
+- Replace the `--proxy` value with your Teleport proxy address or Teleport cloud
+  URI (e.g. `mytenant.teleport.sh:443`):
+
   ```code
   $ teleport db configure create \
     -o file \
-    --proxy=tele.example.com:443 \
+    --proxy=teleport.example.com:443 \
     --token=/tmp/token \
     --azure-postgres-discovery=eastus
   ```
 
 </TabItem>
 <TabItem label="MySQL">
 
+- Specify the region for your database(s) in `--azure-mysql-discovery`.
+- Replace the `--proxy` value with your Teleport proxy address or Teleport cloud
+  URI (e.g. `mytenant.teleport.sh:443`):
+
   ```code
   $ teleport db configure create \
     -o file \
-    --proxy=teleport.example.com:3080 \
+    --proxy=teleport.example.com:443 \
     --token=/tmp/token \
     --azure-mysql-discovery=eastus
   ```
 
 </TabItem>
-</Tabs>
 
-</ScopedBlock>
-<ScopedBlock scope={["cloud"]}>
-
-<Tabs>
-<TabItem label="PostgreSQL">
+<TabItem label="PostgreSQL and MySQL">
 
   ```code
   $ teleport db configure create \
     -o file \
-    --proxy=teleport.example.com:3080 \
+    --proxy=teleport.example.com:443 \
     --token=/tmp/token \
+    --azure-mysql-discovery=eastus \
     --azure-postgres-discovery=eastus
   ```
 
-</TabItem>
-<TabItem label="MySQL">
+<Admonition type="tip">
+This will create two `types` entities in `teleport.yaml`, one for each database
+type. This is useful if you want different regions, tags, or labels for each
+database type.
 
-  ```code
-  $ teleport db configure create \
-    -o file \
-    --proxy=teleport.example.com:3080 \
-    --token=/tmp/token \
-    --azure-mysql-discovery=eastus
-  ```
+Alternatively, you can edit `teleport.yaml` to include both database types in a
+single entry:
+
+```yaml
+db_service:
+  azure:
+  - types: ["mysql", "postgres"]
+  ...
+```
+
+</Admonition>
 
 </TabItem>
-</Tabs>
 
-</ScopedBlock>
+</Tabs>
 
-The command will generate a Database Service configuration with Azure MySQL/Postgres
+This command will generate a Database Service configuration with Azure MySQL/Postgres
 database auto-discovery enabled in the `eastus` region and place it at the
 `/etc/teleport.yaml` location.
 
-### Create a Teleport user
+### Create a Teleport role
+
+On your workstation logged in to your Teleport cluster with `tsh`, define a new
+role to provide access to your Azure database. Our example file is
+`azure-databse-role.yaml`:
+
+```yaml
+version: v6
+kind: role
+metadata:
+  name: azure-database-access
+spec:
+  allow:
+    db_labels:
+      '*': '*'
```

Comment:
```suggestion
      'engine':
        - "Microsoft.DBforMySQL/servers"
        - "Microsoft.DBforMySQL/flexibleServers"
        - "Microsoft.DBforPostgreSQL/servers"
        - "Microsoft.DBforPostgreSQL/flexibleServers"
```
nit: I think we should demonstrate how they can restrict the scope of the access role to just Azure mysql/postgres discovered databases. Otherwise it's a little confusing to call the role "azure-database-access" when it grants access to any database